#!/usr/bin/env python
PACKAGE = "ccrrts"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("GoalSamplingFreq", double_t, 0, "GoalSamplingFreq", 0.0, 0, 1.0)
gen.add("Planner_DT", double_t, 0, "Planner_DT", 0.5, 0, 2.0)
gen.add("MAX_Commited_Length", double_t, 0, "MAX_Commited_Length", 25.0, 0, 50.0)
gen.add("Gamma", double_t, 0, "Gamma", 1.5, 0, 5.0)
gen.add("RiskLimit", double_t, 0, "RiskLimit", 0.5, 0, 1.0)

exit(gen.generate(PACKAGE, "ccrrts", "ccrrtsCTR"))